tests/time_tracker_api/users/users_namespace_test.py

```diff
@@ -2,99 +2,33 @@
 from flask import json
 from flask.testing import FlaskClient
 from flask_restplus._http import HTTPStatus
-from utils.azure_users import AzureConnection
```

Comment:
I have a question if this line `from utils.azure_users import AzureConnection` is deleted, does this line 8 work?  `@patch('utils.azure_users.AzureConnection.get_msal_client', Mock())`

Reply:
Yes, It works. The patch method makes the call to the function.